code/_globalvars/traumas.dm

```diff
@@ -126,9 +126,10 @@ GLOBAL_LIST_INIT(phobia_objs, list(
 		/obj/item/clothing/under/plasmaman/clown
 	)),
 
-	"greytide" = typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/security/cattleprod,
-		/obj/item/spear, /obj/item/clothing/mask/gas, /obj/item/toy/figure/assistant,
-		/obj/structure/statue/sandstone/assistant
+	"greytide" = (typecacheof(list(/obj/item/clothing/under/color/grey, /obj/item/melee/baton/security/cattleprod,
+		/obj/item/spear, /obj/item/toy/figure/assistant,
+		/obj/structure/statue/sandstone/assistant)) + typecacheof(list(/obj/item/clothing/mask/gas), ignore_root_path = FALSE, only_root_path = TRUE
```

Comment:
why do you use typecache of at all, why not just add the path for the gas mask

Reply:
specific check for type in typecache in phobia.dm line 51:
```			if(is_type_in_typecache(O, trigger_objs))```